docs/en/observability/apm/feature-roles.asciidoc

```diff
@@ -9,8 +9,29 @@ NOTE: This documentation only applies to the APM Server binary.
 
 Use role-based access control or API keys to grant APM Server users access to secured resources.
 
-* <<apm-feature-roles>>
-* <<apm-beats-api-keys>>.
+[discrete]
+[[apm-secure-comms-stack-role-based]]
+=== Role-based access
+
+{es-security-features} provides {ref}/built-in-roles.html[built-in roles] that grant a
+subset of the privileges needed by users.
+When possible, assign users the built-in roles to minimize the affect of future changes on your security strategy.
+
+If no built-in role is available, you can assign users the privileges needed to accomplish a specific task by
+creating and assigning feature roles to users.
+
+<<apm-feature-roles,*Read more in Use feature roles →*>>
```

Comment:
Does this framing make sense?

Reply:
It does to me. I'm confused about how to use built-in roles though. We provide
`apm_system` and `apm_user` but I was not able to understand which permissions those would apply and if they were relevant compared to what is written under Use feature roles. I'll do a bit of research there.

Reply:
>Elasticsearch security features provides [built-in roles](https://www.elastic.co/guide/en/elasticsearch/reference/8.15/built-in-roles.html) that grant a subset of the privileges needed by APM users. When possible, assign users the built-in roles to minimize the affect of future changes on your security strategy.

This is in the [current docs](https://www.elastic.co/guide/en/observability/current/apm-secure-comms-stack.html#apm-feature-roles), but if this is no longer relevant we can remove that idea in this PR &mdash; just let me know!

Reply:
Yes, I would remove it. I checked with the team also and those roles are not relevant for the APM Server use cases.

Reply:
```suggestion
[discrete]
[[apm-secure-comms-stack-role-based]]
=== Role-based access

Manage access on a feature-by-feature basis by creating several custom feature-related roles and assigning
one or more of these roles to each APM Server user based on which features they need to access.

<<apm-feature-roles,*Read more in Use feature roles →*>>
```